### PR TITLE
OSDOCS 6914: Updating pod examples to comply with restricted PSA (Sca…

### PR DESCRIPTION
**Pull request by bergerhoffer**
…lability book)

Version(s):
4.15+

Issue:
https://issues.redhat.com/browse/OSDOCS-6914

Link to docs preview:
* https://62529--ocpdocs-pr.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-low-latency-tuning.html#configuring_for_irq_dynamic_load_balancing_cnf-master
* https://62529--ocpdocs-pr.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-low-latency-tuning.html#node-tuning-operator-creating-pod-with-guaranteed-qos-class_cnf-master
* https://62529--ocpdocs-pr.netlify.app/openshift-enterprise/latest/scalability_and_performance/using-cpu-manager.html#seting_up_cpu_manager_using-cpu-manager-and-topology_manager

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by bergerhoffer** [comment]
FYI @sheriff-rh (if you're the right writer for scalability) of this PR to update all pod examples in the scalability book to comply with pod security admission standards. Let me know if you have any questions or concerns on this. Thanks!

**Comment by sheriff-rh** [comment]
Looks great @bergerhoffer, thanks for looping me in! Let me know if I can assist in any way.

File: modules/how-huge-pages-are-consumed-by-apps.adoc [hunk 1]
```diff
@@ -23,6 +23,7 @@ metadata:
   generateName: hugepages-volume-
 spec:
   containers:
+    # TODO: How to update this example since it already has securityContext set
```
Comment (bergerhoffer) [inline modules/how-huge-pages-are-consumed-by-apps.adoc:26]: @s-urbaniak @stlaz Need help with this one too please

Comment (stlaz) [reply modules/how-huge-pages-are-consumed-by-apps.adoc:26]: already privileged, we can leave this one as-is

File: modules/consuming-huge-pages-resource-using-the-downward-api.adoc [hunk 2]
```diff
@@ -26,6 +26,7 @@ metadata:
     app: hugepages-example
 spec:
   containers:
+    # TODO: How to update this example since it already has securityContext set
```
[the remark this answers is not in the document]
Comment (stlaz) [reply modules/consuming-huge-pages-resource-using-the-downward-api.adoc:29]: let's leave it as is

**Comment by bergerhoffer** [comment]
@gkarager Can you please review this from auth perspective?

@mffiedler Can you help with verifying/testing these examples to make sure they still work after these updates? (Or help with who the right person is to help with this?)

Preview links are in the description. Thanks!

**Review by cbippley — COMMENTED** [review]
LGTM!

**Review by skordas — APPROVED** [review]
/LGTM

**Comment by gkarager** [comment]
LGTM from auth perspective, but from the testing perspective **pod is not coming up**.
cc @xingxingxia

**Comment by kalexand-rh** [comment]
The `branch/enterprise-4.15` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.14. And any PR going into `main` must also target the latest version branch (enterprise-4.15).

If the update in your PR does NOT apply to version 4.15 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main

**Comment by kalexand-rh** [comment]
The `branch/enterprise-4.16` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.15. And any PR going into `main` must also target the latest version branch (enterprise-4.16).

If the update in your PR does NOT apply to version 4.16 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Comment by bergerhoffer** [comment]
QE approved above. Merging!

**Pull request merged**